File: Exercícios sobre POO/Empresa 8/Empresa em python/endereco.py
class Endereco:
    def __init__(self, rua, cidade, estado, cep) -> None:
        self.rua = rua
        self.cidade = cidade
        self.estado = estado
        self.cep = cep
    
    def mostrar_endereco(self):
        print("Endereco: ", end="")
        print(f"Rua: {self.rua}, Cidade: {self.cidade}, Estado: {self.estado}, CEP: {self.cep}")

File: Exercícios sobre POO/Empresa 8/Empresa em java/src/Endereco.java
public class Endereco {
    private String rua;
    private String cidade;
    private String estado;
    private String cep;

    public Endereco(String rua, String cidade, String estado, String cep){
        this.rua = rua;
        this.cidade = cidade;
        this.estado = estado;
        this.cep = cep;
    }

    public void mostrar_endereco(){
        System.out.println("Endereco: Rua: " + this.rua + ", Cidade: " +
                this.cidade + ", Estado: " + this.estado + ", CEP: " + this.cep);
    }
}
